libs/newsletter-workflow/src/lib/steps/newsletterData/brazeLayout.ts

```diff
@@ -1,17 +1,33 @@
 import type { WizardStepLayout } from '@newsletters-nx/state-machine';
 import { executeModify } from '../../executeModify';
+import { getStringValuesFromRecord } from '../../getValuesFromRecord';
+import { regExPatterns } from '../../regExPatterns';
 
-const markdownToDisplay = `
+const markdownTemplate = `
 # Modify Braze Values
 
 These are tracking fields used by Braze.
 
-They have been calculated automatically from the Name that you entered on a previous step of the wizard, but you can change them if you need.
+They have been calculated automatically from the name **{{name}}**, but you can change them if you need.
+
+***TO SIMPLY THE DATA COLLECTION PROCESS, THESE WILL BE DISPLAYED READ-ONLY WITH A LINK TO EDIT IF REQUIRED***
 
```

Comment:
The best way to include the values read-only would be to include them in the mark-down. We don't need to link to another page to edit them, it could just be an optional step that include the schema to edit the values ie

```mermaid
flowchart LR
    B[step: Readonly] --> C{buttons}
    C -->|edit button| D[Step: Edit] -->E
    C -->|ok button| E[Step for next question]
```

Reply:
Good idea